### ISSUE TITLE
Build instructions Linux?

### ISSUE THREAD

**Issue by haarts** [nwoltman/srt-to-vtt-converter#1]
It seems this project is only for Mac/Windows? I've been having issues with the compilation on Linux (Ubutnu 14.04). When running make I get:

``` shell
$ make
g++ -std=c++11 -O2 -MMD -I ./include   -c -o Utils.o Utils.cpp
Utils.cpp:16:19: fatal error: codecvt: No such file or directory
 #include <codecvt>
                   ^
compilation terminated. 
make: *** [Utils.o] Error 
```

I've been trying to find the package containing the relevant header file (which is the problem I _think_) and installed;  libboost1.55-dev, libc++-dev,  libstdc++-4.8-dev yet no love. Is this something more fundamental? As you can see I'm not a very good C++ programmer.

**Comment by nwoltman** [comment]
Hi @haarts. I'm assuming you meant to open this in the [srt-to-vtt-cl](https://github.com/woollybogger/srt-to-vtt-cl) repo (no worries though, I know the names are almost exactly the same).

I had tried to build on Linux a while ago and couldn't get it to compile. I believe it requires g++ 5 or clang to compile. I just tried with g++ 5.1.1 and it worked. Here's instructions on how to install g++ 5 on Ubuntu: http://askubuntu.com/questions/618474/how-to-install-the-latest-gcurrently-5-1-in-ubuntucurrently-14-04.
I'll try to add something to the documentation and Makefile on building for Linux soon.
